shaders/src/light_indirect.fs

```diff
@@ -580,7 +580,7 @@ void evaluateIBL(const MaterialInputs material, const PixelParams pixel, inout v
     vec4 Fssr = vec4(0.0f);
     // evaluateScreenSpaceReflections will set the value of ssr if there's a hit.
     // ssr.a contains the reflection's contribution.
-    if (pixel.roughness <= 0.01f && frameUniforms.ssrDistance > 0.0f) {
+    if (pixel.roughness <= 0.1f && frameUniforms.ssrDistance > 0.0f) {
```

Comment:
Could we introduce a #define for this? MAX_SSR_ROUGHNESS or something.

Reply:
no because it's going away soon.